functions/index.js

```diff
@@ -14,6 +15,7 @@ app.use(cors());
 app.use(morgan('dev'));
 //Use Routes Here
 app.use('/v1/auth', users);
+app.use('/v1', routes);
 //Error Handlers
```

Comment:
I think this `routes` should be named descriptively to `locations` for easy identification just like `users`